### DISCUSSION TITLE
I am not able to push to github please help me !!!

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2696] [Q&A]
![Screenshot (67)](https://github.com/user-attachments/assets/0df8cf27-b3fb-45ab-ae94-6a6763645b7d)

**Comment by EngrPips** [comment]
Hello @prakharmishra3956, Pleas watch this lesson https://updraft.cyfrin.io/courses/foundry/foundry-fund-me/pushing-to-github where Patrick taught us how to push our code to Github.

**Comment by Totiruzi** [comment]
@prakharmishra3956 have you been able to solve the issue you are having?
Please better to paste the error in formatted code along with the screenshot 
Thanks )

**Comment by Marvelousy20** [comment]
It looks like you are not connected to the internet.

**Comment by EngrPips** [comment]
Interesting.

**Comment by BowtiedHarpyEagle** [comment]
This looks like you got an Internet problem.

**Comment by EngrPips** [comment]
Indeed.
